src/ui/public/vislib/__tests__/visualizations/tile_maps/map.js

```diff
@@ -16,7 +16,7 @@ import VislibVisualizationsMapProvider from 'ui/vislib/visualizations/_map';
 //   ['rows', require('fixtures/vislib/mock_data/geohash/_rows')],
 // ];
 
-// // TODO: Test the specific behavior of each these
+//
```

Comment:
Does this comment still apply to the commented-out code? If so I think we should leave it. If not then can we delete the commented-out code?